solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -170,6 +178,28 @@ contract WalletRegistry is IRandomBeaconConsumer, Ownable {
         emit RandomBeaconUpgraded(_randomBeacon);
     }
 
+    /// @notice Updates the values of authorization parameters.
+    /// @dev Can be called only by the contract owner, which should be the
+    ///      wallet registry governance contract. The caller is responsible for
+    ///      validating parameters.
+    /// @param _minimumAuthorization New minimum authorization amount
+    /// @param _authorizationDecreaseDelay New authorization decrease delay in
+    ///        seconds
+    function updateAuthorizationParameters(
```

Comment:
We should initialize the values for deployment in the constructor.

Reply:
Good point, see the two recent commits please.